core/pom.xml

```diff
@@ -153,8 +143,12 @@
 			<artifactId>opennlp-tools</artifactId>
 		</dependency>
 		<dependency>
+			<!-- TODO: NemexWekaClassificationEDATest throws "java.lang.Exception: liblinear classes not in CLASSPATH!".
+            This requires a wrapper but the one in https://github.com/bwaldvogel/liblinear-weka is outdated. -->
 			<groupId>de.bwaldvogel</groupId>
 			<artifactId>liblinear</artifactId>
+			<version>2.11</version>
```

Comment:
@madhumita-git can you please verify that this is the correct version? In your commit it was set to 1.5 but versions start at 1.7.1 in Maven Central
Also, build succeeds and all tests pass but NemexWekaClassificationEDATest throws "java.lang.Exception: liblinear classes not in CLASSPATH". Does it happen in your environment as well?